### PR TITLE
Updated migration steps

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

File: docs/pmm-upgrade/migrating_from_pmm_2.md [hunk 1]
```diff
@@ -13,45 +13,100 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
 
 ## Step 2: Migrate PMM 2 Server to PMM 3
 
-Follow these manual steps to upgrade your PMM 2 Server to PMM 3:
-{.power-number}
+=== "For Docker Volume"
```
Comment (Nailya) [inline docs/pmm-upgrade/migrating_from_pmm_2.md:16]: I'd rename it to something like "For those running PMM v2 with docker volumes" to clarify that these steps should be executed if PMM2 server is running using docker volume container

File: docs/pmm-upgrade/migrating_from_pmm_2.md [hunk 2]
```diff
@@ -13,45 +13,100 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
...
+    docker pull percona/pmm-server:3
+    ```
+
+    6. Run new container with existing volume:
+   
+    ```sh
+    docker run -d -v pmm-server-data:/srv -p 80:8080 -p 443:8443 --name pmm-server --restart always percona/pmm-server:3
+    ```
+
+=== "For Data Container"
```
Comment (Nailya) [inline docs/pmm-upgrade/migrating_from_pmm_2.md:58]: Same here. I'd rename it to something like "For those running PMM v2 with docker data containers" to clarify that these steps should be executed if PMM2 server is running using docker data-container

**Review by Nailya — CHANGES_REQUESTED** [review]

File: docs/pmm-upgrade/migrating_from_pmm_2.md [hunk 3]
```diff
@@ -13,45 +13,100 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
...
+    5. Pull PMM 3 Server image:
+   
+    ```sh
+    docker pull percona/pmm-server:3
+    ``` 
+
+    6. Run new container with existing data container:
+
+    ```sh
+    docker run -d --volumes-from pmm-server-data -p 80:8080 -p 443:8443 --name pmm-server --restart always percona/pmm-server:3
```
Comment (ademidoff) [inline docs/pmm-upgrade/migrating_from_pmm_2.md:103]:
```suggestion
    docker run -d --volumes-from pmm-server-data -p 443:8443 --name pmm-server --restart always percona/pmm-server:3
```

File: docs/pmm-upgrade/migrating_from_pmm_2.md [hunk 4]
```diff
@@ -13,45 +13,100 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
...
+    5. Pull PMM 3 Server image:
+
+    ```sh
+    docker pull percona/pmm-server:3
+    ```
+
+    6. Run new container with existing volume:
+   
+    ```sh
+    docker run -d -v pmm-server-data:/srv -p 80:8080 -p 443:8443 --name pmm-server --restart always percona/pmm-server:3
```
Comment (ademidoff) [inline docs/pmm-upgrade/migrating_from_pmm_2.md:55]:
```suggestion
    docker run -d -v pmm-server-data:/srv -p 443:8443 --name pmm-server --restart always percona/pmm-server:3
```

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**